src/main/resources/Quests.json

```diff
@@ -65,6 +66,7 @@
                   "itemQuantity":1
                }
             ],
+            "chunkLocation":null,
```

Comment:
Shouldn't this chunk location be the lumbridge chunk, as that's where you should be to complete the step?

Reply:
Fixed